### ISSUE TITLE
Lack of rules

### ISSUE THREAD

**Issue by johnpauldyar** [wesmurr/MyPlanner#3]
The template set forth by the developers does not have rules about the amount of children a template section could have, beside having at least one or none. It is possible, for example, in the Centre assessment plan that the user should have 3-5 goals. There does not seem to be a way for the developer to allow for that, except by making a template with 3 goals, 4 goals or 5 goals, which seems over the top.

**Comment by wesmurr** [comment]
Yeah we do not enforce a limit. We did not realize that was necessary. We thought it was either going to be only one child or as many as you want. Changing canCopy or canRemove to a min and max limit would be a good idea.
